### PR TITLE
Update non_field_errors object to follow spec

### PR DESCRIPTION
**Pull request by jpadilla**
Went ahead and worked on fixing #113 based on the previous discussion.

### CODE REVIEW

**Comment by dustinfarris** [comment]
This looks good.  Going to try it out before merging.

/cc @holandes22

File: addon/adapters/drf.js [hunk 1]
```diff
@@ -122,12 +122,13 @@ export default DS.RESTAdapter.extend({
...
-              meta: {key: key},
-              detail: error
+              source: { pointer: 'data' },
+              detail: error,
+              title: 'Validation Error'
             });
           } else {
             out.push({
-              source: { pointer: `data/attributes/${key}`},
+              source: { pointer: `data/attributes/${key}` },
```
Comment (dustinfarris) [inline addon/adapters/drf.js]: Ember Data 1.13.7 (released yesterday) "fixed" the pointer format to align with the JSON API spec.

https://github.com/emberjs/data/pull/3558

I think you'll need to add a forward slash to the beginning here (and on line 125), and update the documentation to show compatibility with ED >= 1.13.7

Comment (jpadilla) [inline addon/adapters/drf.js]: @dustinfarris yeah you're about that. Update coming up in a few minutes.

**Comment by jpadilla** [comment]
@dustinfarris updated for ember-data 1.13.7

**Comment by dustinfarris** [comment]
This is working, but ED doesn't seem to do anything with the /data errors.  Going to merge this and open an issue there.

**Comment by dustinfarris** [comment]
Opened https://github.com/emberjs/data/issues/3602

**Comment by jpadilla** [comment]
@dustinfarris awesome, thanks. Subscribed over to that issue for any follow up.

**Pull request merged**